exercises/concept/amusement-park/.docs/instructions.md

```diff
@@ -3,14 +3,14 @@
 It is your job to provide and maintain parts of the IT system of an amusement park.
 In particular, you are responsible for the program that manages visitors and tickets.
 
-First, you will create a new visitor in the system, then you will provide a function for revoking a ticket.
-In task 3 and 4 you will work on the ticket tracking system that helps to prevent forgery.
-Lastly, you will help adapting the system to new legal requirements.
+First, you will create a new visitor in the system. Then you will provide a function for revoking a ticket.
```

Comment:
Might need to put the sentence on a new line. I am not sure. No need to change until clarification is given by @junedev or @SleeplessByte

Reply:
Yes, each sentence should be on a new line (For concept exercise docs, javascript docs. Practice exercise docs are controlled via the problem-specifications repo).

Reply:
Thanks for the thorough review 🙇 

I've inserted new lines in [9c561a8](https://github.com/exercism/javascript/compare/e5415e1ba09ac9b9d3958f6ceb3bfb938a13e4ff..9c561a82bb24ca918392671e371f428525ae37fd). Does it look correct?

Reply:
Looks all good now. I will mark the comments as resolved.